c2/c2-protocol/c2-protocol-api/src/main/java/org/apache/nifi/c2/protocol/api/OperandType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * (c) 2018-2021 Cloudera, Inc. All rights reserved.
```

Comment:
This license header does not meet the standard Apache License 2.0 header guidelines. Please see other files for examples.